overviewer_core/src/composite.c

```diff
@@ -26,13 +26,12 @@
 
 typedef struct {
     PyObject_HEAD
-    Imaging image;
+        Imaging image;
```

Comment:
wat. Am I missing something here? Why is this indented two levels deep and there's nothing after `PyObject_HEAD`?

Reply:
It's probably due to the fact that PyObject_HEAD is a preprocessor that expands to `PyObject ob_base;` according to [this](https://docs.python.org/3/c-api/structures.html).

By adding `;` after it though it seems to fix the indent since it completes the "statement" without clang format thinking you're just using a keyword it doesn't know like putting `unsigned` and `int` on different lines.
```
typedef struct {
    PyObject_HEAD; // < adding a `;` after it fixes this.
    Imaging image;
} ImagingObject;
```